### PR TITLE
added Arbitrary trait (fixes #1)

### LINKED ISSUE #1: Manta structures should have Arbitrary trait implemented
This is useful for testing locally and for other crates that may use the structures in the moray module.

### PR DESCRIPTION
**Pull request by rjloura**

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -5,6 +5,7 @@
 extern crate serde_json;
```
Comment (kellymclaughlin) [inline src/lib.rs]: You can drop this line.

File: src/moray.rs [hunk 2]
```diff
@@ -66,3 +71,77 @@ pub struct MantaDirectory {
     pub roles: Vec<String>, // TODO: double check this is a String
     pub vnode: u64,
 }
+
+// Implement Arbitrary traits for testing
+impl Arbitrary for MantaObjectShark {
+    fn arbitrary<G: Gen>(g: &mut G) -> MantaObjectShark {
+        let len = g.gen::<u8>() as usize;
+        MantaObjectShark {
+            datacenter: util::random_string(g, len),
```
Comment (kellymclaughlin) [inline src/moray.rs:80]: Thinking about the question you asked during office hours Friday, maybe we should make a quickcheck helper crate to share across projects that has this `random_string` function. Another useful function might be one that just generates some arbitrary JSON values.

File: src/moray.rs [hunk 3]
```diff
@@ -66,3 +71,77 @@ pub struct MantaDirectory {
...
+        let mtime: u64 = g.gen();
+        let creator = util::random_string(g, len);
+        let dirname = util::random_string(g, len);
+        let name = util::random_string(g, len);
+        let owner = Uuid::new_v4().to_string();
+        let roles: Vec<String> = vec![util::random_string(g, len)];
+        let vnode: u64 = g.gen();
+        let content_length: u64 = g.gen();
+
+        let md5_sum =
```
Comment (kellymclaughlin) [inline src/moray.rs]: I'd lean on the `Deref` instance here and do this:

```
let md5_sum = md5::compute(util::random_string(g, len));
let content_md5: String = base64::encode(&*md5_sum);
```

File: src/moray.rs (hunk 2 above)
Comment (rjloura) [reply src/moray.rs:80]: Agreed.  Unless there is something that is already part of quickcheck.  Seems odd that there isn't.

**Review by kellymclaughlin — APPROVED** [review]
Looks good!

**Pull request merged**